panel/config.py

```diff
@@ -433,19 +437,29 @@ def __getattribute__(self, attr):
         """
         from .io.state import state
 
-        if attr == '_param__private':
-            return super().__getattribute__('_param__private')
+        # ATTRS[attr]+=1
+        # print(ATTRS)
+        if attr in ('_param__private', '__class__', 'npm_cdn', 'param'):
+            return super().__getattribute__(attr)
 
         # _param__private added in Param 2
         try:
             init = super().__getattribute__('_param__private').initialized
         except AttributeError:
             init = super().__getattribute__('initialized')
-        global_params = super().__getattribute__('_globals')
+
+        if attr=="param":
+            return super().__getattribute__(attr)
+
+        global _PARAMS
+        if not _PARAMS and init:
+            _PARAMS = set(super().__getattribute__('param'))
+
         if init and not attr.startswith('__'):
-            params = super().__getattribute__('param')
```

Comment:
`super().__getattribute__('param')` is a major speed impediment because

- It looks up something that is a constant in my understanding.
- Later we run `__contains__` which is slow because the value is a list.